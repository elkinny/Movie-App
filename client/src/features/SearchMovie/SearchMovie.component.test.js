import SearchMovieComponent from './SearchMovie.component';
import React from 'react';
import { shallow } from './../../test/enzymeSetup.js';

describe('test: ', () => {
  it('SearchMovieComponent to match snapshot', () => {
    const props = {
      handleSubmit: () => {},
      handleToggle: () => {},
      moviesCount: 1,
      movies: [],
      searchToggleValue: 'searchToggleValue',
      sortToggleValue: 'sortToggleValue',
      formComponentRef: () => {},
    };
    const component = shallow(<SearchMovieComponent {...props} />);
    expect(component).toMatchSnapshot();
  });
});
