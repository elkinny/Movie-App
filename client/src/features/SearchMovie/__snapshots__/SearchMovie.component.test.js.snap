// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test:  SearchMovieComponent to match snapshot 1`] = `
<Fragment>
  <SearchFormContainer
    handleSubmit={[Function]}
    handleToggle={[Function]}
    searchToggleValue="searchToggleValue"
  />
  <Navigation>
    <span>
      1 films are found
    </span>
    <Toggle
      labels={
        Array [
          "Rating",
          "Year",
        ]
      }
      onChange={[Function]}
      text="Sort by:"
      type="sort"
      value={true}
    />
  </Navigation>
  <MoviesListComponent
    movies={Array []}
  />
</Fragment>
`;
